cm_sdk/api/__init__.py

```diff
@@ -0,0 +1,4 @@
+__author__ = 'e0d'
+
+class APIException(Exception):
```

Comment:
The common way of doing it is adding a different file and change what gets exposed via `__all__`. See [the documentation](https://docs.python.org/2/tutorial/modules.html#packages) for details.